### PR TITLE
Resources/Documentation Page

### PR DESCRIPTION
**Pull request by exu3**
This PR:
- Removes `Bank Fees` from the navbar
- Replaces "Documents" (legal docs) in the navbar with "Docs" for the documentation/resources page.
  - moves "Documents" (legal docs) to the documentation/resources page
- Adds the documentation (Docs) index page
- Adds 3 article pages which are in `app/controllers/events_controller.rb` for now
Screen Shot 2021-10-13 at 10 05 13 PM
Screen Shot 2021-10-13 at 10 05 20 PM
Screen Shot 2021-10-13 at 10 05 22 PM

### CODE REVIEW

**Comment by MelSmith104** [comment]
@eilla1 amazing job! this looks so good! working on getting you the copy for the check page and setting up a time with zach to review. awesome job!!

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -130,6 +130,114 @@ def card_overview
...
+- If you have your own personal accounts and would prefer to use those rather than Hack Club’s, you may cash out those donations to wherever you’d like; then, simply donate them directly to Hack Club Bank through your donation page or an invoice.
+- To have your payouts sent directly to Hack Club Bank, your fundraising platform of choice must allow for payouts through PayPal. Due to security reasons, we are unable to share our account or routing numbers.
+- *Please keep in mind that PayPal will also charge a fee for your donations which will be.
+
+### Follow these steps to get started!
+
+1. Think of a catchy name for your campaign that fits with your organization’s objective for the fundraiser.
+2. Create an account with the platform you will be using. You can select that you work for a charity as your nonprofit is fiscally sponsored by Hack Club Bank! (Just enter in our EIN 81-2908499).
+3. Then, reach out to the Bank team at [mailto:bank@hackclub.com](bank@hackclub.com) to let us know that you’re creating a fundraising campaign. We’ll help you get set up with payouts through Bank:
```
Comment (garyhtou) [inline app/controllers/events_controller.rb]:
```suggestion
3. Then, reach out to the Bank team at [bank@hackclub.com](mailto:bank@hackclub.com) to let us know that you’re creating a fundraising campaign. We’ll help you get set up with payouts through Bank:
```

**Comment by maxwofford** [comment]
Removing reviewers b/c I did that in person with Mel to show how to review feature works

**Comment by maxwofford** [comment]
@eilla1 i'm going to revert 9df99205 to bring the cards back in a different pr, but i removed them for the time being just to merge this while messing around.

**Comment by maxwofford** [comment]
I'm going to squash this to simplify the merge history– there's something funky going on

**Pull request merged**